### ISSUE TITLE
Is the parameter `derived_params` supposed to be a List type?

### ISSUE THREAD

**Issue by roger-yu-ds** [datajoely/modular-spaceflights#4]
The input is:
```yml
  derived:
    - column_a: number_of_reviews
      column_b: total_fleet_count
      numpy_method: divide
      conjunction: over # {column_a}_{over}_{column_b}
    - column_a: number_of_reviews
      column_b: total_fleet_count
      numpy_method: divide
      conjunction: over
    - column_a: reviews_per_month
      column_b: company_rating
      numpy_method: multiply
      conjunction: by
```

So the type should be `List[Dict[str, str]]` instead of just `Dict[str, str]`, right?

**Comment by datajoely** [comment]
Yes that's a mistake on my part!

**Comment by datajoely** [comment]
Closed by #5
